api/services/managedInstance/aws/parameters/mi-id.yaml

```diff
@@ -1,4 +1,4 @@
-in: query
+in: path
```

Comment:
Did you verify that all consumers of `mi-id.yaml` intended for it to be a path parameter?

Reply:
Yes